polkadot/runtime/parachains/src/configuration.rs

```diff
@@ -261,6 +261,8 @@ pub struct HostConfiguration<BlockNumber> {
 	/// The minimum number of valid backing statements required to consider a parachain candidate
 	/// backable.
 	pub minimum_backing_votes: u32,
+	/// Client features enablement.
+	pub client_features: ClientFeatures,
```

Comment:
In fact it is part of the `HostConfiguration` which is already session buffered. So caching by SessionIndex should work already.